alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -104,8 +110,10 @@ class AlertIndices(
     @Volatile private var historyEnabled = AlertingSettings.ALERT_HISTORY_ENABLED.get(settings)
 
     @Volatile private var historyMaxDocs = AlertingSettings.ALERT_HISTORY_MAX_DOCS.get(settings)
+    @Volatile private var findingMaxDocs = AlertingSettings.ALERT_FINDING_MAX_DOCS.get(settings)
 
     @Volatile private var historyMaxAge = AlertingSettings.ALERT_HISTORY_INDEX_MAX_AGE.get(settings)
+    @Volatile private var findingMaxAge = AlertingSettings.ALERT_HISTORY_FINDING_MAX_AGE.get(settings)
 
     @Volatile private var historyRolloverPeriod = AlertingSettings.ALERT_HISTORY_ROLLOVER_PERIOD.get(settings)
```

Comment:
Copy `historyEnabled` and `historyRolloverPeriod` for the findings index and create settings for them.